### DISCUSSION TITLE
Can the 2.1 models created be used in automatic1111

### DISCUSSION THREAD

**Discussion by shyt47** [TheLastBen/fast-stable-diffusion#958] [Q&A]
Can the 2.1 models created be used in automatic1111 webui?

**Comment by TheLastBen** [comment]
Yes

**Reply by shyt47** [reply]
I'm getting an error when trying to load the 2.1 model I just trained.
raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for LatentDiffusion:
        size mismatch for model.diffusion_model.input_blocks.1.1.proj_in.weight: copying a param with shape torch.Size([320, 320])

**Reply by TheLastBen** [reply]
Use the latest colab, remove the folder SD and make sure you choose the correct version in the last cell.

**Reply by shyt47** [reply]
was getting that error in automatic webui. I figured out I needed to download the .yaml file,  now no error but i get a black . image.  tried both versions of the yaml

**Reply by TheLastBen** [reply]
in my colab you don't need to download any yaml file, as long as you use the latest version of the notebook.

**Reply by shyt47** [reply]
I use your colab to train the model, but run automatic1111 locally. so your saying I shouldn't need the yaml? because without it  i get errors, with I get only black images.

**Reply by SAC020** [reply]
In automatic1111 you definitely need the yaml, renamed to match your model name.

mymodel.ckpt <=> mymodel.yaml

Without the yaml, auto1111 will not load the model

**Reply by shyt47** [reply]
OK yep. Thought so. Not sure what went wrong. Trained 2.1 model with 521. Tried both Yaml files. only black images for results. I have a 3080 non TI
